src/__tests__/fileMock.ts

```diff
@@ -0,0 +1 @@
+module.exports = "div";
```

Comment:
what is this for?

Reply:
Ah, I'll write a comment for this file. This is used only by the unit tests to mock assets. If an svg is loaded, instead of jest dealing with the big svg file, it'd mock it with a `div`. So you'd get something that looks weird like `<div {...svgProps} >` instead of `<svg {...svgProps}>`.